float64 x
float64 y
float64 xDot
float64 yDot
float64 psi
float64 psiDot